types/react-native-canvas/index.d.ts

```diff
@@ -176,8 +176,8 @@ export class Image {
 export class ImageData {
     constructor(canvas: Canvas, data: number[], height: number, width: number);
     readonly data: number[];
-    readonly height: number;
-    readonly width: number;
+    readonly height: number | undefined;
```

Comment:
Please provide documentation why these can be undefined. If these can be undefined, does it mean constructor can be invoked without height and width being passed or could it pass undefined (optional or required parameter accepting undefined)